### DISCUSSION TITLE
what is equivelent in JSON of password = {include "fileTopwdPath" } in auth

### DISCUSSION THREAD

**Discussion by amcharhal** [nanomq/nanomq#1730] [General]
I'm using JSON to configure a NanoMQ broker, and I couldn't find a way to replace {include "filePath"}. I tried password:{include: "pathToFile"}, but it's not working

**Comment by StargazerWayne** [comment]
In your case, conf should be like this `password = {include "pathToFile"}`. And make sure you got the `pathToFile` right. For more details, you can refer to our [official docs](https://nanomq.io/docs/en/latest/config-description/acl.html).
